modules/EPMC.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+import logging
+
+import pyspark
+from pyspark.sql.types import StringType
+from pyspark.sql import SparkSession 
+import pyspark.sql.functions as pf
+from pyspark import SparkContext
+from pyspark.conf import SparkConf
```

Comment:
Alphabetical order as per PEP8

```suggestion
import argparse
import logging
import sys

import pyspark
from pyspark import SparkContext
from pyspark.conf import SparkConf
from pyspark.sql import SparkSession 
from pyspark.sql.types import StringType
import pyspark.sql.functions as pf
```